src/systems/userinput/devices/app-aware-touchscreen.js

```diff
@@ -278,8 +299,17 @@ export class AppAwareTouchscreenDevice {
 
     if (hasCursorJob) {
       const assignment = findByJob(MOVE_CURSOR_JOB, this.assignments);
+
       frame[path.isTouchingGrabbable] = assignment.framesUntilGrab <= 0;
       assignment.framesUntilGrab--;
+
+      if (assignment.framesUntilUnassign >= 0) {
```

Comment:
Odd that until touch end, this check is `false` because it's asking `undefined >= 0`